network.tf

```diff
@@ -0,0 +1,35 @@
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.prefix}-rg"
+  location = "West Europe"
+}
+
+resource "azurerm_virtual_network" "vnet" {
+  name                = "${var.prefix}-virtual-network"
+  address_space       = ["10.0.0.0/16"]
```

Comment:
It would be better to move this value to variables

Reply:
fixed